src/app/icd/ICDNotifier.h

```diff
@@ -104,6 +110,7 @@ class ICDNotifier
     void BroadcastActiveRequestNotification(ICDListener::KeepActiveFlags request);
     void BroadcastActiveRequestWithdrawal(ICDListener::KeepActiveFlags request);
     void BroadcastICDManagementEvent(ICDListener::ICDManagementEvents event);
+    void BroadcastSubscriptionReport();
```

Comment:
Broadcast is an unfortunate name. Notify would have been better since there is no "broadcasting" in the networking/protocol sense happening here.

Reply:
renamed all of the to Notify*